push_ghrc.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+echo $PAT | docker login ghcr.io --username $GIT_USERNAME --password-stdin
+docker tag $timestamp_holder ghrc.io/$GIT_REPO
+docker image push ghrc.io/$GIT_REPO
+
+#docker rmi $latest_holder
+#docker rmi $timestamp_holder
```

Comment:
The acronym in the file name is not right. Could you please correct it to reflect the "container registry" (`ghcr`) correctly? Thanks.

Reply:
Sorry about that. This is done now.